src/main/java/edu/bu/metcs673/trackr/transaction/TransactionController.java

```diff
@@ -60,7 +60,7 @@ public ResponseEntity<GenericApiResponse<List<Transaction>>> findAllTransactionB
 	 * @author Xiaobing Hou
 	 * @date 05/22/2022
 	 */
-	@GetMapping("/find/{id}/{bankAccountId}")
+	@GetMapping("/{bankAccountId}/{id}")
```

Comment:
Changing that order is going to have unintended consequences on the frontend if we don't update it as well.

Reply:
i did this to match the same format as the "findAll" method.  I felt that it would help make the app more consistent.